app/src/main/java/com/andryoga/safebox/ui/common/Utils.kt

```diff
@@ -23,4 +23,8 @@ object Utils {
     fun setTextViewLeftDrawable(view: TextView, drawableId: Int) {
         view.setCompoundDrawablesWithIntrinsicBounds(drawableId, 0, 0, 0)
     }
+
+    fun switchText(view: TextView, changeToText: String) {
+        view.text = changeToText
```

Comment:
This method was supposed to work this way:
if view's current text is "originalText" then change it to "changeToText" text otherwise change it to "originalText". That's why the name is **switch**Text.

Reply:
okay I will implement that